to-be-ccpp-ized/utilities/vertical_diffusion/vertical_diffusion_solver.F90

```diff
@@ -0,0 +1,136 @@
+module vertical_diffusion_solver
+
+    implicit none
+    private
+    save
```

Comment:
Could I ask if this `SAVE` is needed here? It seems like there are no module-level variables in `vertical_diffusion_solver`.

Reply:
That's good to know, thanks @gold2718!

Reply:
Removed!

Reply:
Also, because there is no private data in these modules, is the `private` even necessary?  I only copied that over because I've seen that in other modules so it looks similar but if it's not explicitly needed, happy to remove it as well.